test/project/config/default/kustomizeconfig.yaml

```diff
@@ -0,0 +1,38 @@
+# This file is static and is generated by kubebuilder
+nameReference:
```

Comment:
Put the following into `cert-manager` base
```
nameReference:
- kind: Issuer
  group: certmanager.k8s.io
  fieldSpecs:
  - kind: Certificate
    group: certmanager.k8s.io
    path: spec/issuerRef/name
```

Put the following into the `web-hook` base
```
nameReference:
- kind: Service
  version: v1
  fieldSpecs:
  - kind: CustomResourceDefinition
    group: apiextensions.k8s.io
    path: spec/conversion/webhookClientConfig/service/name
  - kind: MutatingWebhookConfiguration
    group: admissionregistration.k8s.io
    path: webhooks/clientConfig/service/name
  - kind: ValidatingWebhookConfiguration
    group: admissionregistration.k8s.io
    path: webhooks/clientConfig/service/name
```